extensions-core/namespace-lookup/src/main/java/io/druid/server/namespace/cache/NamespaceExtractionCacheManager.java

```diff
@@ -21,10 +21,7 @@
 
```

Comment:
Moving to `io.druid.query.lookup.namespace` and `io.druid.server.lookup.namespace` as appropriate